src/main/check.cpp

```diff
@@ -57,6 +57,7 @@ int main(int argc, char** argv) {
     app.add_flag("-s", ebpf_verifier_options.strict, "Apply additional checks that would cause runtime failures");
     app.add_flag("-v", verbose, "Print both invariants and failures");
     app.add_flag("--no-simplify", ebpf_verifier_options.no_simplify, "Do not simplify");
+    app.add_flag("-p", ebpf_verifier_options.print_line_info, "Print line information");
```

Comment:
I'm not sure the letter is indicative. Maybe `n`? Or simply a longer flag such as `--line-info`.